teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; pocket_size: iterable of float or int
&nbsp; &nbsp; &nbsp; &nbsp; Lengths of edges defining the binding site.
&nbsp; &nbsp; num_poses: int
&nbsp; &nbsp; &nbsp; &nbsp; Maximum number of poses to generate.
</pre><p> </p><ul><li>Above, see <strong>edges </strong>and <strong>maximum</strong>.</li><li>Use <code>subprocess.check_output</code>  instead of the <code>!xxx</code>  magic, because there's a chance our readers want to use this outside Jupyter. </li><li>Return the output.</li><li>Do not join the options list. Actually, do not build two different lists. Just use "command" and define it at once (no need for multiple +=).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>